libraries/lambda_handlers/convert_handler.py

```diff
@@ -36,4 +42,22 @@ def _handle(self, event, context):
         converter = self.converter_class(source=source, resource=resource, cdn_file=cdn_file, options=options)
         results = converter.run()
         converter.close()  # do cleanup after run
+        if callback is not None:
+            self.callback_payload = json.loads(json.dumps(self.data))  # clone so we can modify
```

Comment:
I think deepcopy() is a better way to clone. See my data_utils.py for how I do it to mask sensitive fields that we log.